### ISSUE TITLE
glTF2 Assets not loading

### ISSUE THREAD

**Issue by nshelton** [googlevr/poly-toolkit-unity#22]
I have noticed that recently many assets do not load (see below). This is in a fresh project using Unity 2017.3, only importing the Poly toolkit and nothing else. This is both through the C# API and Unity Inspector GUI.

This is caused by an early out in `ImportGltf.cs` function 

```  // Annotates GltfPrimitive with materials, precursorMeshes
  static List<MeshPrecursor> CreateMeshPrecursorsFromPrimitive(ImportState state, GltfPrimitiveBase prim) {
    if (prim.mode != GltfPrimitiveBase.Mode.TRIANGLES) {
      Debug.LogWarningFormat("Cannot create mesh from {0}", prim.mode);
      return null;
    }
...
```

In the case of these models below, `prim.mode` is `0`, which has no corresponding enum (The only value is `TRIANGLES=4`  ).

For these models, if I comment out the `return null;` there, the model successfully imports.

Also note that all the models that failed to import were glTF2, maybe GltfPrimitiveBase.Mode doesnt get set in that case.

Example Broken Assets:

hammer 
https://poly.google.com/view/cOizz1RJnb3
parrot 
https://poly.google.com/view/dfNjMLtO0pd 
elephant 
https://poly.google.com/view/9J_cG39KYFC
gun
https://poly.google.com/view/aLbeOSoOpXN
hand 
https://poly.google.com/view/btWmPNVSKUc
gun
https://poly.google.com/view/ap9I6sv347V
tank
https://poly.google.com/view/4t0RMXCl_Ud
...

I am sure there are many more, almost all models I tried from User https://poly.google.com/user/4aEd8rQgKu2 would not import into Unity.

**Comment by anokta** [comment]
The issue should be fixed now via [v1.2.2](https://github.com/googlevr/poly-toolkit-unity/releases/tag/v1.1.2) (see 6a11f4aa264f6161f39730ffdc5068f2fb58e726 for the relevant change).

**Comment by nshelton** [comment]
awesome thanks!
